api/scpca_portal/utils.py

```diff
@@ -50,3 +54,15 @@ def filter_dict_list_by_keys(list_of_dicts, included_keys, *, ignore_value_error
         new_list_of_dicts.append(dictionary_copy)
 
     return new_list_of_dicts
+
+
+def write_dict_list_to_file(
+    list_of_dicts: List[Dict],
+    output_file_name: str,
+    field_names: Set,
+    delimiter: str,
```

Comment:
We can do that. I thought it might be beneficial to have a more generic utils function which could write any csv-like file, but if you think that our only use case is for writing `tsv` files, then I'm with you. I'll remove the delimiter argument and default to a tab delimiter when configuring the  `DictWriter`.